config/database.yml

```diff
@@ -11,14 +11,18 @@ default: &default
 
```

Comment:
- [ ] Kindly make sure to use `postgresql` as the database, for now, your using `sqlite`